airbyte-webapp/src/packages/cloud/views/DefaultView.tsx

```diff
@@ -1,12 +1,24 @@
+import { useEffect } from "react";
 import { Navigate } from "react-router-dom";
 
+import { useExperiment } from "hooks/services/Experiment";
+
 import { RoutePaths } from "../../../pages/routePaths";
 import { CloudRoutes } from "../cloudRoutes";
+import { EXP_SOURCE_SIGNUP_SELECTOR } from "../components/experiments/constants";
 import { useListCloudWorkspaces } from "../services/workspaces/CloudWorkspacesService";
 
 export const DefaultView: React.FC = () => {
   const workspaces = useListCloudWorkspaces();
+  // exp-signup-selected-source-definition
+  const isSignupSourceSelectorExperiment = useExperiment("authPage.signup.sourceSelector", false);
+  const sourceDefinitionId = localStorage.getItem(EXP_SOURCE_SIGNUP_SELECTOR);
 
+  useEffect(() => {
+    setTimeout(() => {
+      localStorage.removeItem(EXP_SOURCE_SIGNUP_SELECTOR);
```

Comment:
remove localStorage item as we don't need it anymore

Reply:
What is the reason to wrap this in a 5s delay here? (Also if we want to have this wrapped, me must make sure to clear the setTimeout in case the component gets unmounted in the meantime to prevent memory leaks)

Reply:
You are right, I don't think we need it after thinking about it again. I will remove it.